css/_secondarybar.scss

```diff
@@ -0,0 +1,76 @@
+@import "./mixin";
+.secondarybar-container {
+  width: 100vw;
+  position: fixed;
+  display: flex;
+  justify-content: flex-end;
+}
+.secondarybar {
+  @include flexcenter-row;
+
+  width: 94.8%;
+  height: 59px;
+  float: right;
+  position: relative;
+  z-index: -1;
+  background-color: $grey-bg;
+  border-top: 1px solid lighten($grey, 15);
+  border-bottom: 1px solid lighten($grey, 15);
```

Comment:
if you are using lighten with a value often, it's better to make it a new color variable